IDEAS/Buildings/Data/Insulation/Glasswool.mo

```diff
@@ -1,16 +1,12 @@
 within IDEAS.Buildings.Data.Insulation;
 record Glasswool = IDEAS.Buildings.Data.Interfaces.Insulation (
-    k=0.040,
-    c=840,
-    rho=80) "Glasswool" annotation (Documentation(info="<html>
-<p>
-Glass wool insulation thermal properties.
-</p>
+    k=0.035,
```

Comment:
Interesting, I thought that 0.04 was more accurate but commercial data sheets seem to confirm this value.